mlrun/api/utils/projects/member.py

```diff
@@ -30,21 +29,9 @@ def ensure_project(
             format_=mlrun.api.schemas.ProjectsFormat.name_only,
             leader_session=auth_info.session,
         )
-        if name in project_names.projects:
-            return False
-        logger.info(
-            "Ensure project called, but project does not exist. Creating", name=name
-        )
-        project = mlrun.api.schemas.Project(
-            metadata=mlrun.api.schemas.ProjectMetadata(name=name),
-        )
-        self.create_project(
-            db_session,
-            project,
-            leader_session=auth_info.session,
-            wait_for_completion=wait_for_completion,
-        )
-        return True
+        if name not in project_names.projects:
+            raise mlrun.errors.MLRunNotFoundError(f"Project {name} does not exist")
+        return False
```

Comment:
The bool this function returned was used to determine whether it led to a project creation, I believe we don't need to return it anymore